pensieve/experimenter.py

```diff
@@ -16,10 +16,11 @@ class Variant:
 
 @attr.s(auto_attribs=True)
 class Experiment:
-    slug: str
+    slug: str  # experimenter slug
     start_date: Optional[dt.datetime]
     end_date: Optional[dt.datetime]
     variants: List[Variant]
+    normandy_slug: Optional[str] = attr.ib(default=None)
```

Comment:
nit (I think this should work?):

```suggestion
    normandy_slug: Optional[str] = None
```